hazelcast/src/test/java/com/hazelcast/test/mocknetwork/MockConnectionManager.java

```diff
@@ -286,22 +285,21 @@ private boolean send(Packet packet, Address target, SendTask sendTask) {
 
     private final class SendTask implements Runnable {
 
+        private final AtomicInteger retries = new AtomicInteger();
```

Comment:
`retries` was read twice and written once in the `run()` method. An `AtomicInteger` should perform better and needs no FB suppression.

Reply:
Performance in such situation will hardly be an issue.